### DISCUSSION TITLE
Github pages

### DISCUSSION THREAD

**Discussion by airdward** [community/community#30321] [Other Feature Feedback, Questions, & Ideas]
Hi guys, quite a minor issue for y'all but quite an impasse for me, how do I go past this
Error: Error: No uploaded artifact was found! Please check if there are any errors at build step, or uploaded artifact name is correct.

**Comment by airtower-luna** [comment]
Please post a link to the build log (or repository, if you have trouble finding the log) so people can actually take a look. Without that information it's impossible to say why that might be happening.

**Reply by airdward** [reply]
Here it is
https://github.com/iamoghenerukevwe/personal

**Reply by radar-blip** [reply]
Your index.html isn't quite right, you need more in it than what there is. Dont know if that will fix the problem though.

**Reply by airtower-luna** [reply]
There seems to be some weird stuff going on with cancelled builds, too. Did you cancel any of them? If not that kind of looks like a bug to me.
